website/pages/en/developing/creating-a-subgraph.mdx

```diff
@@ -281,6 +281,7 @@ We support the following scalars in our GraphQL API:
 | `Int8` | An 8-byte signed integer, also known as a 64-bit signed integer, can store values in the range from -9,223,372,036,854,775,808 to 9,223,372,036,854,775,807. Prefer using this to represent `i64` from ethereum. |
 | `BigInt` | Large integers. Used for Ethereum's `uint32`, `int64`, `uint64`, ..., `uint256` types. Note: Everything below `uint32`, such as `int32`, `uint24` or `int8` is represented as `i32`. |
 | `BigDecimal` | `BigDecimal` High precision decimals represented as a significand and an exponent. The exponent range is from −6143 to +6144. Rounded to 34 significant digits. |
+| `Timestamp` | Scalar for `timestamp` fields for timeseries and aggregations. |
```

Comment:
This looks like it's the timestamp change from #683? may want to rebase from Master

Reply:
i have never done this. I was looking on google on how to do this but got super confused, can you help me with this?

Reply:
All good, I rebased for now but can walk you through it at a good moment